spec/requests/api/providers_spec.rb

```diff
@@ -438,6 +455,19 @@ def token(connection)
     end
   end
 
+  describe "custom attributes assignment" do
+    it "custom attributes assignment" do
+      api_basic_authorize collection_action_identifier(:providers, :update_custom_attributes)
+      p1 = FactoryGirl.create(:ext_management_system, :name => "provider")
+      run_post(providers_url, gen_request(:update_custom_attributes, custom_attributes_request))
```

Comment:
Updating the provider's attributes based on the provider name seems like it could be brittle, and not consistent with our other APIs.

Perhaps this could be part of the regular update for provider resources, i.e.

```
POST /api/providers/<provider id>

~or~

POST /api/providers

{
  "action": "edit"
  "resources": [
    {"id": <provider id>, "custom attributes": {}}
  ]
}
```